### PR TITLE
Enable Distributed Tracing acceptance tests chain in openshift-e2e-test-qe-chain

### PR DESCRIPTION
**Pull request by IshwarKanse**
* Enable Distributed Tracing acceptance tests in OCP CI jobs.

### CODE REVIEW

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-ipi-workers-rhel8-f14

**Comment by IshwarKanse** [comment]
/pj-rehearse pull-ci-openshift-distributed-tracing-qe-main-ocp4.13-amd64-acceptance-distributed-tracing-tests-acceptance

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-ipi-workers-rhel8-f14

**Comment by IshwarKanse** [comment]
@liangxia Could you help to review this change, thanks.

File: ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ chain:
   - ref: cucushift-e2e
   - ref: openshift-extended-web-tests
   - ref: openshift-extended-test-supplementary
+  - chain: distributed-tracing-tests-common
```
Comment (liangxia) [inline ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml:13]: @jhou1 Adding the chain here means we will run those tests run as part of OCP tests. Is it expected ?

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-distributed-tracing-qe-main-ocp4.13-amd64-acceptance-distributed-tracing-tests-acceptance

File: ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml (hunk 1 above)
Comment (wzheng1) [reply ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml:13]: If possible, we can have a meeting to discuss why we add this chain, by this time, we can consider this scenario with CY24 O1KR1.

**Comment by IshwarKanse** [comment]
Closing this for now.

**Pull request closed without merging**